internal/config/env.go

```diff
@@ -6,6 +6,17 @@ import (
 
 type Environment map[string]string
 
+// Set will only set the given environment key and value if it isn't already set
+// within the Environment.
+func (e *Environment) Set(key, value string) {
+	env := *e
```

Comment:
Use directly, what's the purpose of this...

Reply:
Need to do this for accessing index, otherwise I get a NonIndexableOperand error.

Reply:
Fixed with magic :P